includes/amp-post-template-functions.php

```diff
@@ -18,6 +18,7 @@ function amp_post_template_init_hooks() {
 	add_action( 'amp_post_template_head', 'amp_add_generator_metadata' );
 	add_action( 'amp_post_template_head', 'wp_generator' );
 	add_action( 'amp_post_template_head', 'amp_post_template_add_block_styles' );
+	add_action( 'amp_post_template_head', 'amp_post_template_add_default_styles' );
```

Comment:
Shouldn't the block styles come after the default styles?

Reply:
They could, but they have until now been coming before.

Reply:
@schlessera My thinking here was to minimize the diff in the CSS for now.

Reply:
Yes, if it already was in that order, that's fine with me.